flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/CreditBasedSequenceNumberingViewReader.java

```diff
@@ -45,6 +47,8 @@
 
 	private final PartitionRequestQueue requestQueue;
 
+	private final int initialCredit;
```

Comment:
Renaming it to a boolean type `withoutExclusiveCredits` seems more direct to understand